### PR TITLE
{devel}[fosscuda/2018b] ZeroMQ v4.2.5

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2431.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/042ea7a50b3383060342e987eeed8241 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2016.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/aa9cdf4a629b2f58807506801e7f46b6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node3118.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/b8acda1d059bd42cbf71e5ee2963064f for a full test report.

File: easybuild/easyconfigs/z/ZeroMQ/ZeroMQ-4.2.5-fosscuda-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+checksums = ['cc9090ba35713d59bb2f7d7965f877036c49c5558ea0c290b0dcc6f2a17e489f']
+
+# --with-pgm will use shipped OpenPGM (in foreign subdir)
+configopts = '--with-pic --with-system-pgm '
+configopts += 'OpenPGM_CFLAGS="-I$EBROOTOPENPGM/include/pgm-${EBVERSIONOPENPGM%.*}" '
+configopts += 'OpenPGM_LIBS="-L$EBROOTOPENPGM/lib -lpgm -lrt -lpthread -lm" '
+
+dependencies = [
+    ('OpenPGM', '5.2.122'),
+    ('util-linux', '2.32.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/z/ZeroMQ/ZeroMQ-4.2.5-fosscuda-2018b.eb]: @akesandgren This needs to be downgraded to `2.32` to make the dependency variants test pass...

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/4127ab85f093a75a0a72cb7cecdc53e9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3105.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/56299ef5d6497dcc270c69fb98484fb8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/164176037842bc35e1be6fbcd333c4c4 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**